brewtils/models.py

```diff
@@ -1649,3 +1651,60 @@ def __repr__(self):
             self.storage,
             self.details,
         )
+
+
+class TopicSubscribers:
+    schema = "TopicSubscribersSchema"
+
+    def __init__(self, topic=None, subscribers=[]):
+        self.topic = topic
+        self.subscribers = subscribers
```

Comment:
Purely a style comment, in our other models if there is a list, we set the default to None. Then on the assign we do:

`self.subscribers = subscribers or []`

Reply:
Fixed